docs/components/connectors/out-of-the-box-connectors/available-connectors-overview.md

```diff
@@ -4,38 +4,59 @@ title: Overview
 description: Take a closer look at the Connectors available in Camunda Platform 8.
 ---
 
-This section gives an overview of the **out-of-the-box Connectors** available in Camunda Platform 8. All Connectors are available for Camunda Platform 8 SaaS and [Self Managed](../../../self-managed/connectors-deployment/install-and-start.md).
+Out-of-the-box Connectors accelerate solution implementation by providing pre-built, ready-to-use connectors to popular external systems.
```

Comment:
Would likely note the abbreviation of OOTB here too, given we use it in screenshots.

Reply:
Great spot, will note.